src/types.d.ts

```diff
@@ -0,0 +1,18 @@
+type ResponseDataType = {
```

Comment:
It seems to me that we can improve readability by omitting the "Type" from our type names. It should be just `ResponseData` and `User`. Variables and Types live in a different namespace, meaning that we can have both

```js
type User = { firstName: string }
const User: User = { firstName: 'Someone' }
```